control/controlclient/direct.go

```diff
@@ -459,8 +463,10 @@ func (c *Direct) doLogin(ctx context.Context, t *oauth2.Token, flags LoginFlags,
 	c.mu.Unlock()
 
 	if err != nil {
+		systemd.Status("login failed")
```

Comment:
Rather than in this file, you probably want to put these in ipn/local.go's LocalBackend.enterState method.

And include which email address you're authenticated as (`b.activeLogin`).